### PR TITLE
Create estimation.md

### LINKED ISSUE #77: Determine how we'll do estimation
### Goals
* To align as a team on how we'll do estimation for GH issues and milestones so that we can better plan our scope for each cycle and our overall roadmap.

### Tasks

- [x] Determine when and whether we'll use T-shirt sizes or Points to estimate issues and milestones
- [x] If Points, determine whether we'll use a Fibonacci scale (1, 2, 3, 5, 8, 13) or an exponential scale (1, 2, 4, 8, 16, 32)
- [ ] Discuss our interest in measuring velocity and burndown.  
- [ ] If interested, discuss our process for reviewing velocity and burndown.
- [ ] Align on a reference story for a mid-sized issue
- [ ] Document decisions and processes
- [x] Provide Sheila some background info on points and estimation

### PR DESCRIPTION
**Pull request by alexsoble**
# Notes 

+ [Preview link](https://github.com/18F/OPRE-Unicorn/blob/ars/77/add-estimation-md/docs/how_we_work/estimation.md). 👁️ 
+ Closes #77.

### CODE REVIEW

**Review by carjug — APPROVED** [review]
LGTM! Thanks, Alex.

**Review by amymok — APPROVED** [review]
LGTM!

File: docs/how_we_work/estimation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+We will use points for estimation, with an exponential scale: 1, 2, 4, 8, 16, 32.
+
+Reference points for what each level means, via [@amymok](https://github.com/18F/OPRE-Unicorn/issues/77):
+
+Points | Meaning
+---|---
+1 | Really trivial. Typos, word changes.
+2 | Easy to change. No need to do additional research. Requires a bit more effort than a trivial task. Examples: Fix a linter error. Fix a single unit test. Reword paragraphs. 
+4 | Requires more effort. A medium task. May some research on reminding yourself how to do a task, but no unknowns.
```
Comment (ninamak) [inline docs/how_we_work/estimation.md:15]: Typo fix.  Change "May some research" to "May involve some research" or "May require some research"

**Review by ninamak — APPROVED** [review]
LGTM!  One small typo to fix and then good to merge @alexsoble

File: docs/how_we_work/estimation.md (hunk 1 above)
Comment (alexsoble) [reply docs/how_we_work/estimation.md]: Good catch, thanks @ninamak!

**Pull request merged**